src/endpoints/brands/brandBySlug.ts

```diff
@@ -8,6 +8,6 @@ export function createBrandBySlugEndpointRequest(
 ): BapiCall<BrandBySlugEndpointResponseData> {
   return {
     method: 'GET',
-    endpoint: `brands/${slug}`,
+    endpoint: `brands/${slug}?forceSlug=true`,
```

Comment:
Please use the params object here and not the endpoint which is basically the `path`

Reply:
Moved `forceSlug` as the part of `params`